app/views/users/show.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="container">
+<div>
+<div class="image">Images</div>
+<div class="card">
+<h1 class="header"><%=@user.name%></h1>
+<p class="footer">Number of posts: <%=@user.posts_counter %></p>
+</div>
+</div>
+<div class="bio">
+<p>Bio:</p>
+<p><%=@user.bio%></p>
+</div>
+<% @user.recent_posts.each do |post| %>
+<div class="post">
+<p><%=post.title %></p>
```

Comment:
- [ ] Why dont you link the post title to the post-show page like this :point_right: `<h4><%= link_to post.title, user_post_path(post) %></h4>` :blush: